### PR TITLE
Add setup instructions

### PR DESCRIPTION
**Pull request by ErinBecker**
Based on setup section of our [AAS workshop website](https://abostroem.github.io/2021-01-07-aas-online/) and [setup instructions](https://datacarpentry.org/ecology-workshop/setup-python-workshop.html) for the Python section of the Data Carpentry Ecology workshop. I've simplified the instructions in two major ways:

1) Removing instructions on using Miniconda (which is an option in the Ecology setup instructions).
2) Removing instructions for installing libraries in an existing Conda environment (which is listed as not recommended on the AAS workshop website).

I still need to:
- [ ] Add information about the data
- [x] Upload the revised test_setup.ipynb file, which I have generalized from the version @AllenDowney created for the AAS workshop.

### CODE REVIEW

File: code/environment.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  - numpy
+  - matplotlib
+  - seaborn
+  - pandas
+  - pytables
+  - h5py
+  - scipy
+  - astropy
+  - astroquery
+  - gsl
```
Comment (rudyphd) [inline code/environment.yml:18]: I'm curious about how we are using gsl? Is this a dependency from one of the other tools?

Comment (AllenDowney) [reply code/environment.yml:18]: I don't think we're using it. I'm not sure where that line came from, but I don't think it was me.

Comment (rudyphd) [inline code/environment.yml:18]:
```suggestion

```

**Review by rudyphd — APPROVED** [review]
Removed gsl dependency, not used explicitly.

**Comment by ErinBecker** [comment]
I'm merging on approval from @rudyphd. Will need to put in a separate PR once we have a link to the data files.

**Pull request merged**